addons/optics/XEH_postInit.sqf

```diff
@@ -3,8 +3,18 @@
 
 if (!hasInterface) exitWith {};
 
+#define ALL_CAMERAS [ \
```

Comment:
Can be replaced by `EFUNC(isFeatureCameraActive)` and on line 17 simply check boolean return.